### PR TITLE
Update the support-bundle-image setting

### PR DESCRIPTION
**Pull request by w13915984028**
Add more details about support-bundle-image setting and suggest user not to set it in general.

Issue: https://github.com/harvester/harvester/issues/6277

### CODE REVIEW

File: docs/advanced/settings.md [hunk 1]
```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
...
+
+:::note
+
+- Harvester always sets the `default` field to the image tag which is packed into Harvester ISO. When the cluster is upgraded, the `default` field is also updated automatically. As long as the `value` field is not set or empty, Harvester sticks to the `default` field. It makes sure the image is on the cluster and is up-to-date.
+
+- In general you do not need to set this setting. If you plan to set a specific image, be aware that it will take precedence of the `default` and is not updated automatically.
+
+- If you have set a customized image but want to follow the default image, then use the `kubectl` command to clear the `value` field.
+
+- The UI menu **Use the default value** is not the best option to clear this setting. It copies the current `default` to `value`, the `default` will be updated when upgrading but the `value` field will not.
```
Comment (w13915984028) [inline docs/advanced/settings.md:516]: cc @torchiaf @a110605 @bk201 

It is a bit surprised to observe the side affect of `Use the default value` in this context. UI? copies the `default` field to `Value` field.

Alternatively, if user clicks `Use the default value` and does not further change the configuration, when `saving`, UI deletes the `value` field; thus it really always sticks to `default`.

In different Harvester releases, Harvester may change the `default`.

WDYT? thanks.

**Review by FrankYang0529 — APPROVED** [review]
LGTM.

File: docs/advanced/settings.md [hunk 2]
```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
...
+...
+status: {}
+value: '{"repository":"rancher/support-bundle-kit","tag":"v0.0.38","imagePullPolicy":"IfNotPresent"}'  // copied from default
+```
+
+In the future when you upgrade the cluster to a new version the `default` image tag may be like `v0.0.40`, the `value` field will be outdated again.
+
+:::note
+
+- Harvester always sets the `default` field to the image tag which is packed into Harvester ISO. When the cluster is upgraded, the `default` field is also updated automatically. As long as the `value` field is not set or empty, Harvester sticks to the `default` field. It makes sure the image is on the cluster and is up-to-date.
```
Comment (jillian-maroket) [inline docs/advanced/settings.md:510]:
```suggestion
- The value of `tag` in the `default` field is always based on the image that is packed into the Harvester ISO. This field is automatically updated whenever the cluster is upgraded. 

- The `default` field is used when the `value` field is not set or is left empty. Harvester checks if the default image is stored in the cluster and is up-to-date.
```

File: docs/advanced/settings.md [hunk 3]
```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
...
+
+- In general you do not need to set this setting. If you plan to set a specific image, be aware that it will take precedence of the `default` and is not updated automatically.
```
Comment (jillian-maroket) [inline docs/advanced/settings.md:512]:
```suggestion
- Configuring this setting is not required. If you decide to specify a different image tag in the `value` field, remember that this tag may become outdated when the cluster is upgraded.
```

File: docs/advanced/settings.md [hunk 4]
```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
...
+
+- If you have set a customized image but want to follow the default image, then use the `kubectl` command to clear the `value` field.
```
Comment (jillian-maroket) [inline docs/advanced/settings.md:514]:
```suggestion
- Use the command `$ kubectl edit settings.harvesterhci.io support-bundle-image` to clear the `value` field.
```

File: docs/advanced/settings.md (hunk 1 above)
Comment (jillian-maroket) [inline docs/advanced/settings.md:516]:
```suggestion
- The **Use the default value** button on the Harvester UI only copies the contents of the `default` field to the `value` field. You may use this to replace an outdated image tag, but the copied tag will eventually become outdated as well (when the cluster is upgraded and the `default` field is updated).
```

File: docs/advanced/settings.md [hunk 5]
```diff
@@ -424,6 +426,99 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
...
+
+- The UI menu **Use the default value** is not the best option to clear this setting. It copies the current `default` to `value`, the `default` will be updated when upgrading but the `value` field will not.
+
+- If your cluster is in an air-gapped environment and the set `support-bundle-image` is not the default one, you need to prepare this image on your local [containerd registry](#containerd-registry). The operation [Generate a Support Bundle](../troubleshooting/harvester.md#generate-a-support-bundle) will fail if the set image is not available.
```
Comment (jillian-maroket) [inline docs/advanced/settings.md:518]:
```suggestion
- If your cluster is in an air-gapped environment and you specified a non-default image tag in the `value` field, ensure that the image is available in the local [containerd registry](#containerd-registry). Harvester is unable to [generate a support bundle](../troubleshooting/harvester.md#generate-a-support-bundle) if the image is not available.
```

**Review by jillian-maroket — COMMENTED** [review]
Review done. The draft was difficult to understand so I rewrote everything.

**Comment by w13915984028** [comment]
@jillian-maroket The PR is updated with your comments and rebased to master-head, please take a new look, thanks.

**Review by jillian-maroket — APPROVED** [review]

**Review by bk201 — APPROVED** [review]

**Pull request merged**